extras/vagrantExample/src/main/vagrant/Vagrantfile

```diff
@@ -200,74 +247,93 @@ EOF
+	fi
 	echo "Sesame workbench deployed at http://rya-example-box:8080/openrdf-workbench"
 
-	echo "Downloading Rya"
-	# Right now it's on dropbox, but eventually it'll be on maven...
-
-	download -L https://www.dropbox.com/s/7e74yiuq4jmu0od/rya.indexing.example-${RYA_EXAMPLE_VERSION}-distribution.zip?dl=0 -o rya.indexing.example-${RYA_EXAMPLE_VERSION}-distribution.zip
-	sudo mkdir rya.indexing.example-${RYA_EXAMPLE_VERSION}-distribution
-	sudo unzip rya.indexing.example-${RYA_EXAMPLE_VERSION}-distribution.zip -d rya.indexing.example-${RYA_EXAMPLE_VERSION}-distribution
+	echo "Installing Rya"
+	ryaIndexing=rya.indexing.example-${RYA_EXAMPLE_VERSION}-distribution
+	if [[ ! -s ${ryaIndexing}.zip ]] ; then
+		# Right now it's on dropbox, but eventually it'll be on maven...
+		echo "Downloading quietly, this will take some minutes with no output..."
+		download --output ${ryaIndexing}.zip https://dl.dropbox.com/s/7e74yiuq4jmu0od/${ryaIndexing}.zip?raw=1  || exit 112
+	fi
+	sudo mkdir --parents ${ryaIndexing}
+	sudo unzip -q -o ${ryaIndexing}.zip -d ${ryaIndexing}
 	
+    # before continueing, wait for tomcat to deploy wars:
+    waitForDeploy /var/lib/tomcat7/webapps/openrdf-workbench/WEB-INF/lib/
+    waitForDeploy /var/lib/tomcat7/webapps/openrdf-sesame/WEB-INF/lib/
+
 	# soft linking the files doesn't seem to work in tomcat, so we copy them instead :(
-	sudo cp rya.indexing.example-${RYA_EXAMPLE_VERSION}-distribution/dist/lib/* /var/lib/tomcat7/webapps/openrdf-workbench/WEB-INF/lib/
-	sudo cp rya.indexing.example-${RYA_EXAMPLE_VERSION}-distribution/dist/lib/* /var/lib/tomcat7/webapps/openrdf-sesame/WEB-INF/lib/
+	sudo cp ${ryaIndexing}/dist/lib/* /var/lib/tomcat7/webapps/openrdf-workbench/WEB-INF/lib/ || exit 113
+	sudo cp ${ryaIndexing}/dist/lib/* /var/lib/tomcat7/webapps/openrdf-sesame/WEB-INF/lib/    || exit 114
 
 	# These are older libs that breaks tomcat 7
-	sudo rm /var/lib/tomcat7/webapps/openrdf-workbench/WEB-INF/lib/servlet-api-2.5.jar
-	sudo rm /var/lib/tomcat7/webapps/openrdf-workbench/WEB-INF/lib/jsp-api-2.1.jar
-	sudo rm /var/lib/tomcat7/webapps/openrdf-sesame/WEB-INF/lib/servlet-api-2.5.jar
-	sudo rm /var/lib/tomcat7/webapps/openrdf-sesame/WEB-INF/lib/jsp-api-2.1.jar
+	sudo rm --force /var/lib/tomcat7/webapps/openrdf-workbench/WEB-INF/lib/servlet-api-2.5.jar
+	sudo rm --force /var/lib/tomcat7/webapps/openrdf-workbench/WEB-INF/lib/jsp-api-2.1.jar
+	sudo rm --force /var/lib/tomcat7/webapps/openrdf-sesame/WEB-INF/lib/servlet-api-2.5.jar
+	sudo rm --force /var/lib/tomcat7/webapps/openrdf-sesame/WEB-INF/lib/jsp-api-2.1.jar
 	
 	sudo chown -R tomcat7:tomcat7 /var/lib/tomcat7/webapps/openrdf-workbench/WEB-INF/lib/
 	sudo chown -R tomcat7:tomcat7 /var/lib/tomcat7/webapps/openrdf-sesame/WEB-INF/lib/
 
-	# Download and install new templates for OpenRdf WorkBench
-	download -L -O https://www.dropbox.com/s/dgw63m66nubyy4z/rya.vagrant.example-${RYA_EXAMPLE_VERSION}.jar
```

Comment:
We made the switch to Accumulo 1.7.1 in previous update for [RYA-49] (https://issues.apache.org/jira/browse/RYA-49) and RYA-61 because [1.6.4 was missing] (http://apache.mirrors.pair.com/accumulo/1.6.4/) .  I think we [intended to switch]  (https://github.com/apache/incubator-rya/pull/27)  to  1.6.5, but bwmcbride made some improvements and bumped it to 1.7.1 .  It all worked so we took [his contribution] (https://github.com/apache/incubator-rya/pull/34).  
If you think we should go to 1.6.5 I can do it.  The current development pom.xml references 1.6.4  .  I wonder if it is no longer supported and that is why the tarball is missing.